### PR TITLE
nvram: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/nvram.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- [d]elete a firmware variable:
+
+`sudo nvram -d {{name}}`
+
+- [c]lear all the firmware variables:
+
+`sudo nvram -c`
+
+- Set firmware variable from a specific [x]ML [f]ile:
```
Comment (sbrl) [inline pages/osx/nvram.md:26]: Perhaps this will read better?
```suggestion
- Set a firmware variable from a specific [x]ML [f]ile:
```

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

Just a teensy comment below.

File: pages/osx/nvram.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- [p]rint all the variables stored in the NVRAM:
+
+`nvram -p`
+
+- [p]rint all the variables stored in the NVRAM using [x]ML format:
+
+`nvram -xp`
+
+- Set a firmware variable:
```
Comment (marchersimon) [inline pages/osx/nvram.md:14]: Does `name` have to exist or can you create new variables? If not, how about
```suggestion
- Modify the value of a firmware variable:
```

**Review by marchersimon — APPROVED** [review]
Looks good

**Pull request merged**